src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Shape {
+    private int hypotenuse;
```

Comment:
do we need this field?

Reply:
Yes I think that we have to have the available  to comply with the conditions of task. Namely (each figure has color and **unique property**). Hypotenuse is unique property of right triangle.